contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
 import { ComparisonHeader } from 'components/ComparisonTable/header'
 import { ComparisonRow } from 'components/ComparisonTable/row'
 
-Want to know how PostHog and Amplitude compare? If you remember nothing else, remember these two things:
+Amplitude and PostHog are superficially similar. 
 
-1. **Amplitude** is a product analytics tool built for non-technical users and data analysts.
+They both offer product analytics and a range of additional tools, like feature flags, A/B testing, and session replay. 
 
-2. **PostHog** is an all-in-one platform built for engineers that replaces Amplitude _and_ numerous other tools.
+But, deep down, Amplitude is an analytics tool with some extra bits grafted on. In contrast, PostHog is built from the ground up as an all-in-one platform for building successful products. Here's how they compare.
 
 ## How is PostHog different?
 
-### 1. We're everything you need in one
+### 1. We're engineering and product-led
 
-PostHog replaces multiple tools in your data stack. You could run Amplitude for analytics, [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags and testing, [Hotjar](/blog/posthog-vs-hotjar) for session replays and surveys, or you could just use PostHog for everything. One platform, one bill, and seamless integration.
+What does this mean?
 
-### 2. We're built for engineers
+- It means engineers make product decisions.
+- It means our pricing is 100% transparent.  
+- It means they do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
+- It means [we ship fast](/changelog) and iterate based on user feedback. 
+- It means we work in the open – [our roadmap](/roadmap) is public, as is [our company handbook](/handbook). 
+- It means start using PostHog without ever jumping on a quick call – see [how we do sales](/sales).
 
-We support all popular (and some unpopular) [SDKs](/docs/libraries), have [extensively documented APIs](/docs/api), and a [powerful SQL query builder](/docs/hogql), so you can analyze data how you want. We make it easy to [test in production](/product-engineers/testing-in-production), conduct [phased rollouts](/tutorials/phased-rollout), run [beta programs](/tutorials/public-beta-program), and so much more.
+### 2. We're building a genuine all-in-one platform
 
-### 3. We're open source and transparent
+We want you to spend less time engineering your data and more time engineering your product. You can ditch [Mixpanel](/blog/posthog-vs-mixpanel) and Amplitude for analytics, [Hotjar](/posthog-vs-hotjar) for replays and surveys, and [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags and experiments, with one seamlessly integrated platform. Need a data warehouse as well? We've [got you covered](/data-warehouse).
```

Comment:
ON the surface Amplitude is also all-in-one. The paragraph didn't really tell me how we're different